js/flightlog.js

```diff
@@ -1147,7 +1147,16 @@ FlightLog.prototype.rcCommandRawToThrottle = function(value) {
 
 FlightLog.prototype.rcMotorRawToPct = function(value) {
     // Motor displayed as percentage
-    return Math.min(Math.max(((value - this.getSysConfig().motorOutput[0]) / (this.getSysConfig().motorOutput[1] - this.getSysConfig().motorOutput[0])) * 100.0, 0.0),100.0);
+    let motorPct;
```

Comment:
Why are we changing this - we are sending the motor limits explicitly to be able to use the correct values and not having to make assumptions.

Reply:
Which assumptions are you referring to?

Reply:
All that are made in this proposal.

Reply:
That's not actually helping us get anywhere now is it :-)

Reply:
I've been using this build for about 200 or more log analyses now.
In day to day use, focusing on idle and motors, it seems perfect to me.
So if you are proposing a change, then please be constructive and indicate what exactly you think should be changed.